frontend/src/metabase/admin/datamodel/components/database/MetadataTableList.jsx

```diff
@@ -37,44 +52,33 @@ export default class MetadataTableList extends Component {
     });
   }
 
+  renderTable = table => (
```

Comment:
`renderX` methods tend to be a bit of a weird pattern. Could we move this to just be a `<TableRow />` component instead?